code-gurus/besty
# Language Screen

Resolves #4  

# How can be tested:
- Clone the repo and check out ft/language-screen
-  Install the dependencies: `npm install`
- Start the project by running: `expo start`
- Press `a` for IOS or `i` for Android to start the project

Using the expo client (physical device):
- Login with email and password
- Navigate to profile, and choose press on the app to open it. 

<img width="415" alt="Screen Shot 2020-02-13 at 17 54 00" src="https://user-images.githubusercontent.com/20681465/74452965-71cdfd80-4e8a-11ea-9f24-3e5ef89b4953.png">

![Screenshot_1581609355](https://user-images.githubusercontent.com/20681465/74452977-75fa1b00-4e8a-11ea-9acf-5910d9423820.png)
